DepositeCalcTests/Pages/SettingsPage.cs

```diff
@@ -0,0 +1,57 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DepositeCalcTests.Pages
+{
+    internal class SettingsPage
+    {
+        private readonly IWebDriver driver;
+        public SettingsPage(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+        private IWebElement DateFormatFld => driver.FindElement(By.XPath("//select[@id='dateFormat']"));
```

Comment:
Please add a blank line between constructor and properties.
```suggestion
        }
        
        private IWebElement DateFormatFld => driver.FindElement(By.XPath("//select[@id='dateFormat']"));
```